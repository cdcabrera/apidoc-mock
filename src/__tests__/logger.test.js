const logger = require('../logger');

describe('logger', () => {
  it('should return specific properties', () => {
    expect(Object.keys(logger)).toMatchSnapshot('specific properties');
  });
});
