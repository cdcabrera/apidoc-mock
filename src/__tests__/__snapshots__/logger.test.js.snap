// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logger should return specific properties: specific properties 1`] = `
[
  "logger",
]
`;
